libs/styles/variables.css

```diff
@@ -104,6 +104,11 @@
   --grid-margins-width-10: calc((100vw - (var(--grid-column-width) * 10)) / 2);
   --grid-margins-width-8: calc((100vw - (var(--grid-column-width) * 8)) / 2);
   --grid-margins-width-6: calc((100vw - (var(--grid-column-width) * 6)) / 2);
+
+  /* icon sizes */
+  --l-icon: 56px;
+  --m-icon: 40px;
+
```

Comment:
We should follow the same naming convention as other size vars.  elem+attr+unit
`--icon-size-m`